.github/workflows/build-sdks.yml

```diff
@@ -103,12 +103,11 @@ jobs:
 
       - name: Install deps
         run: |
-          codename=$(lsb_release -c | awk '{print $2}')
-          echo "deb http://mirror.genesisadaptive.com/ubuntu/ $codename main" > ./gitaction.list
-          echo "deb http://mirror.math.princeton.edu/pub/ubuntu/ $codename main" ./gitaction.list
-          echo "deb http://mirror.pit.teraswitch.com/ubuntu/ $codename main" >> ./gitaction.list
-          sudo mv -f ./gitaction.list /etc/apt/sources.list.d/
-      
+          #codename=$(lsb_release -c | awk '{print $2}')
+          #echo "deb http://mirror.genesisadaptive.com/ubuntu/ $codename main" > ./gitaction.list
+          #echo "deb http://mirror.math.princeton.edu/pub/ubuntu/ $codename main" ./gitaction.list
+          #echo "deb http://mirror.pit.teraswitch.com/ubuntu/ $codename main" >> ./gitaction.list
+          #sudo mv -f ./gitaction.list /etc/apt/sources.list.d/
```

Comment:
Can we remove commented code

Reply:
Nope just leave them as in case if they are needed in future they can enabled back